docs: last bits of the revamp project (#1713)
NVIDIA/NeMo-Guardrails: docs/configure-rails/caching/index.md

# Caching Instructions and Prompts

The NVIDIA NeMo Guardrails library provides two caching strategies to reduce inference latency.
The in-memory model cache stores LLM responses and returns them for repeated prompts without calling the LLM again.

> KV cache reuse is a NIM-level optimization that avoids recomputing decoder layers for the shared system prompt on each NemoGuard NIM call.

Reviewer: ```suggestion
KV cache reuse is a NIM-level optimization that avoids computation of the system prompt on each NemoGuard NIM call.
```
